lib/ReactViews/FeatureInfo/FeatureInfoDownload.jsx

```diff
@@ -14,7 +14,7 @@ import Styles from './feature-info-download.scss';
 const FeatureInfoDownload = createReactClass({
     propTypes: {
         data: PropTypes.object.isRequired,
-        name: PropTypes.string.isRequired,
+        baseFilename: PropTypes.string.isRequired,
```

Comment:
Ah thanks, had not realised this could create a breaking change for upstream projects, have reverted.